### ISSUE TITLE
Incoming messages are only allowed in Api inboxes

### ISSUE THREAD

**Issue by sinapse-i-a** [chatwoot/chatwoot#6066]
I'm trying to send a new message to a created conversation, but it keeps showing this error.

**Scenario 1:**
a - User sends message via Whatsapp. 
b - I check if the contact exists, 
c - I check if there is a conversation. 
d - Send new message to chatwoot agent.

But I'm always getting a 422 status with the following message: Incoming messages are only allowed in Api inboxes

Payload:
`{
    content: 'Hello',
    content_type: 'text',
    private: false, 
    message_type: 'incoming',
    content_attributes: {}
}`

**Comment by pranavrajs** [comment]
@sinapse-i-a I don't understand the use case. Are you looking to build some integration on the WhatsApp channel? Would you be able to share more details?

**Comment by sinapse-i-a** [comment]
> @sinapse-i-a I don't understand the use case. Are you looking to build some integration on the WhatsApp channel? Would you be able to share more details?

I'm developing a back-end to use other whatsapp brokers, but when I was sending the message to Chattwoot it was showing a 404 error (Create contact, create conversation, create message).

However, analyzing the comments, I managed to understand that for this case I would have to create an Inbox of the API type. After making this configuration and generating a token, I was able to send the conversations. Thanks for listening
